### DISCUSSION TITLE
Change the way no entry signs work

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17722] [Feature requests]
In Parkitect, when a guest is leaving the park, they ignore no entry signs. Could this feature be brought to RCT2 so that people couldn't trap guests in their park?

**Comment by ocalhoun6** [comment]
Could be problematic because one possible use of the 'no entry' signs is to create a system of 'staff only' paths (such as emulating the tunnels under Disney parks). But with this change, peeps who are going home would start using the staff only paths, which kind of ruins it.

Another important use of 'no entry' signs is to block off a path you plan to delete, so that you can delete it without leaving a lot of guests wandering around on the grass (or worse, falling into water). But if peeps going home could ignore the signs, then your soon-to-be-deleted path could still have a constant flow of peeps on it, so you'll never be able to safely delete it.

Having the park rating quickly tank if you block the entrance and trap guests is probably fine.

And even if you did this change, there are still many ways to trap guests. The simplest would be to simply delete the path to the entrance. If you wanted to trap guests while still accepting new guests into the park, you could use a ride with an entrance accessible to the park entrance, but the ride exit _not_ accessible to the park entrance. Or you could mess with a path's connection flags using the tile inspector, so peeps only see it as connected in one direction.

I think the current solution of lost guests causing a park rating crash (and most scenarios requiring a certain park rating to win) is a much more elegant solution, because it discourages guest-trapping behavior no matter _what_ creative way you come up with to trap them.

**Comment by Broxzier** [comment]
Why should it ever be okay for guests to not obey a no-entry sign? If the player is abusing it to keep guests inside their park, their park rating already drops, so this feature will be useless for balancing.

**Comment by karst** [comment]
it's the entire fun of the no entry sign to trap your guests at the entrance, build water under the entrance and remove the paths. 

And for more realistic reasons I often use No-Entry signs to create one way roads (exits to rides, two wide roads where people walk on the correct side of the road instead of through eachother, etc) 

If you want people to ignore your no entry sign, just make a banner saying "No Entry", make it red, and they dont listen to it.

**Comment by 733737** [comment]
In Parkitect, it seems to balance just fine; and if people don't like it maybe there could be a toggle in the options menu.

**Reply by karst** [reply]
You can just write it on a banner, no need for change. It would break many existing parks aswell.
